### PR TITLE
add: swagger add token

### PR DESCRIPTION
**Pull request by rmzf9192**
pass swagger request api

### CODE REVIEW

**Comment by SbloodyS** [comment]
@rmzf9192 Hi, please associate the issue and describe why you want to do that.

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/SwaggerConfig.java [hunk 1]
```diff
@@ -81,4 +89,21 @@ private ApiInfo apiV2Info() {
                 .version("V2")
                 .build();
     }
+
+    private List<Parameter> setHeaderToken() {
+        List<Parameter> pars = new ArrayList<>();
+        // token请求头
```
Comment (EricGao888) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/SwaggerConfig.java:95]: Could you please change this comment into English? Thanks.

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/SwaggerConfig.java [hunk 2]
```diff
@@ -81,4 +89,21 @@ private ApiInfo apiV2Info() {
...
+        String testTokenValue = "";
+        ParameterBuilder tokenPar = new ParameterBuilder();
```
Comment (EricGao888) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/SwaggerConfig.java:97]: What about changing `tokenPar` to `parameterBuilder` for better readability?

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/SwaggerConfig.java [hunk 3]
```diff
@@ -81,4 +89,21 @@ private ApiInfo apiV2Info() {
                 .version("V2")
                 .build();
     }
+
+    private List<Parameter> setHeaderToken() {
+        List<Parameter> pars = new ArrayList<>();
```
Comment (EricGao888) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/SwaggerConfig.java:94]: What about changing `pars` to `headerTokens` for better readability?

**Comment by fuchanghai** [comment]
i want to know why you want to do that

**Comment by davidzollo** [comment]
hi, thanks for your first contribution.
Is there an issue to describe the problem that this PR will solve?

**Pull request closed without merging**